app/models/event.rb

```diff
@@ -9,13 +9,23 @@ class Event < ApplicationRecord
   validate :no_collision
 
   def start_time_must_be_lower_than_end_time
-    errors.add(:start_time, "Can't be higher than end_time") if start_time.try(:>=, end_time)
+    errors.add(:start_time, "Start time must be lower than end time") if start_time.try(:>=, end_time)
   end
 
   def no_collision
     return unless start_time && end_time
-    return unless Event.in_span_for_conference_room(start_time, end_time, conference_room).exists?
-    errors.add(:start_time, "Can't start while another event is in progress")
+    event = Event.in_span_for_conference_room(start_time, end_time, conference_room).first
```

Comment:
would it make sense to do:
`conference_room.events.in_span(start_time, end_time)` ?